.github/workflows/question-closer.yml

```diff
@@ -0,0 +1,23 @@
+name: Question Closer
+on:
+  schedule:
+    - cron: 20 11 * * * # Run at 11:20 AM UTC (3:20 AM PST, 4:20 AM PDT)
+  workflow_dispatch:
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        uses: actions/checkout@v2
+      - name: Install Actions
+        run: cd ./.github/actions && npm install --production && cd ../..
+      - name: Stale Closer
+        uses: ./.github/actions/StaleCloser
+        with:
+          labels: question
+          ignoredLabels: debugger
```

Comment:
Why would the debugger issues be ignored?

Reply:
Bob asked for debugger issues to be ignored.  But, we can ask Andrew if he wants to opt in to the policies.